src/registrar/tests/test_transition_domain_migrations.py

```diff
@@ -607,7 +631,7 @@ def compare_tables(
         total_domain_informations = len(DomainInformation.objects.all())
         total_domain_invitations = len(DomainInvitation.objects.all())
 
-        print(
+        logger.debug(
```

Comment:
This one too since we're here - I think this was supposed to be removed but never was